### PR TITLE
Librdkafka snappy

### PR DESCRIPTION
**Pull request by rotrida**

### CODE REVIEW

**Comment by myd7349** [comment]
According the log, the failure is caused by the `WITH_SSL` this time. It is defined as:
https://github.com/edenhill/librdkafka/blob/d443b4f025058172d2c8568376e36e54aba0b8b4/src/win32_config.h#L36

so we should not turn off `WITHOUT_WIN32_CONFIG` (My fault again). So I think your original change is good.

File: ports/librdkafka/CONTROL [hunk 1]
```diff
@@ -18,3 +18,7 @@ Build-Depends: zlib
 Feature: zstd
 Description: Build with zstd
 Build-Depends: zstd
+
+Feature: snappy
+Description: Build with snappy
+Build-Depends: snappy
```
Comment (myd7349) [inline ports/librdkafka/CONTROL]: Please consider remove this line.

File: ports/librdkafka/portfile.cmake [hunk 2]
```diff
@@ -17,6 +17,7 @@ vcpkg_check_features(
     ssl WITH_SSL
     zlib WITH_ZLIB
     zstd WITH_ZSTD
+	snappy WITH_SNAPPY
```
Comment (myd7349) [inline ports/librdkafka/portfile.cmake]: Please consider using spaces instead of tabs here to keep consistency.

**Review by myd7349 — CHANGES_REQUESTED** [review]

File: ports/librdkafka/CONTROL [hunk 3]
```diff
@@ -1,5 +1,5 @@
 Source: librdkafka
-Version: 1.1.0
+Version: 1.1.1
```
Comment (myd7349) [inline ports/librdkafka/CONTROL]: Bump the version number as 1.1.0-1.

File: ports/librdkafka/CONTROL (hunk 1 above)
Comment (rotrida) [reply ports/librdkafka/CONTROL]: Sure. I will change it.

**Comment by wangli28** [comment]
Related issue: #7454

**Pull request merged**